server/jetstream_api.go

```diff
@@ -146,6 +144,21 @@ const (
 	// JetStreamAdvisoryConsumerMaxDeliveryExceedPre is a notification published when a message exceeds its delivery threshold.
 	JSAdvisoryConsumerMaxDeliveryExceedPre = "$JS.EVENT.ADVISORY.CONSUMER.MAX_DELIVERIES"
 
+	// JSAdvisoryStreamCreatedT notification that a stream was created
+	JSAdvisoryStreamCreatedT = "$JS.EVENT.ADVISORY.STREAM.CREATED"
```

Comment:
T suffix usually preserved for templates that have %s or %d's in them.

Reply:
Should these have stream and optionally consumer in the subject for authorization?

Reply:
I use them like this `subj := JSAdvisoryConsumerDeletedT + "." + o.stream + "." + o.name`, so not printf style but still not the full end result, can make it sprintf if you prefer

Reply:
Let's do that for consistency, also more efficient probably garbage wise..